substrate/client/cli/src/commands/run_cmd.rs

```diff
@@ -149,36 +149,44 @@ pub struct RunCmd {
 	pub keystore_params: KeystoreParams,
 
 	/// Shortcut for `--name Alice --validator` with session keys for `Alice` added to keystore.
-	#[arg(long, conflicts_with_all = &["bob", "charlie", "dave", "eve", "ferdie", "one", "two"])]
+	#[arg(long, conflicts_with_all = &["alice", "bob", "charlie", "dave", "eve", "ferdie", "one", "two"])]
```

Comment:
@iStrike7  is there a reason to keep old names? suchh alice, bob,...?

Reply:
No reason that I can think of rn, but I'd rather not remove them preemptively... Unless you want me to for some reason